src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -122,15 +123,15 @@ public User updateUserFieldsById(UserUpdateFields userUpdateFields, Integer user
       // Handle Roles
       if (Objects.nonNull(userUpdateFields.getUserRoleIds())) {
         List<Integer> currentRoleIds = userRoleDAO.findRolesByUserId(userId).stream()
-            .map(UserRole::getRoleId).collect(Collectors.toList());
+            .map(UserRole::getRoleId).toList();
```

Comment:
The toList changes make the value returned immutable so that we preserve the database being the source of truth.